File: docs/components_page/components/button/active_disabled.jl
using DashBootstrapComponents

buttons = html_div([
    dbc_button("Regular", color = "primary", className = "me-1"),
    dbc_button("Active", color = "primary", active = true, className = "me-1"),
    dbc_button("Disabled", color = "primary", disabled = true),
]);

File: docs/components_page/components/button/active_disabled.R
library(dashBootstrapComponents)

buttons <- div(
  list(
    dbcButton("Regular", color = "primary", className = "me-1"),
    dbcButton("Active", color = "primary", active = TRUE, className = "me-1"),
    dbcButton("Disabled", color = "primary", disabled = TRUE)
  )
)
